README.md

```diff
@@ -0,0 +1,57 @@
+<div align="center">
+  <img src="https://text.media.giphy.com/v1/media/giphy.gif?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJrZXkiOiJwcm9kLTIwMjAtMDQtMjIiLCJzdHlsZSI6InJhZ2UiLCJ0ZXh0IjoiR3JvdXAlMjAyNCIsImlhdCI6MTczNTIwMzkwMX0.6dBXa0DW7iOQGETxKrxHG39dDYceUY57aFiex_UZ6LM" alt="Demo GIF">
+</div>
+<!-- MD03/no-inline-html: Inline HTML is used intentionally to center the image for better visual alignment. -->
+
+# This Project is All About Teamwork! 🤝
+
+Together, we dive into coding and testing challenges, sharpening our problem-solving and collaboration skills as a united team. Our mission is to work in harmony, review each others solutions, and grow through shared learning and constructive feedback. We believe that exceptional solutions stem from exceptional teamwork. That's why we are dedicated to building and supporting each other every step of the way. Let's make amazing things happen, together! ⭐
+
+---
+
+## 🚀 Project Overview
+
+### What We will Do
+
+- 🧠 **Coding Challenges**: We will face a variety of coding problems, each offering new learning opportunities.
+- 🔍 **Review Each Others Solutions**: After completing tasks, we will give and receive feedback to ensure high-quality results and growth.
+- 🛠️ **Collaborate on Testing**: Testing is a key part of the process. We will work together to ensure our solutions are robust and reliable.
+- 💬 **Communicate Effectively**: From code discussions to problem-solving, clear and open communication is key to our success.
+- 🚧 **Solve Roadblocks Together**: No one is left behind. If a challenge arises, we will come together to brainstorm and troubleshoot.
+- 🌱 **Learn and Grow**: The goal is not just to finish the tasks, but to grow as a team through shared knowledge and experiences.
+
+---
+
+## 🌟 How We Collaborate
+
+🧩 We take collaboration seriously. Here is how we do it:
+
+- [Our Norms](https://github.com/MIT-Emerging-Talent/ET6-foundations-group-24/blob/main/collaboration/README.md)
+- [Our Communication](https://github.com/MIT-Emerging-Talent/ET6-foundations-group-24/blob/main/collaboration/communication.md)
+- [Our Constraints](https://github.com/MIT-Emerging-Talent/ET6-foundations-group-24/blob/main/collaboration/constraints.md)
+- [Our Learning Goals](https://github.com/MIT-Emerging-Talent/ET6-foundations-group-24/blob/main/collaboration/learning_goals.md)
+- [Our Retrospectives](https://github.com/MIT-Emerging-Talent/ET6-foundations-group-24/blob/main/collaboration/retrospective.md)
+- [Our challenges](https://github.com/MIT-Emerging-Talent/ET6-foundations-group-24/tree/main/solutions)
+
+---
+
+## Meet the Dream Team 🧑‍🤝‍🧑
+
+Our squad is made up of some seriously talented people who bring the 🔥 to this project. Check out the crew making it all come to life:
+
+- **Abel Mesfin Teka**
+- **Banchiamlak**
+- **Gai Samuel**
+- **Khusro Sakhi**
```

Comment:
I don't know khusros but you can ask him , i think he is a student at computer science